src/lib/components/MobileNav.svelte

```diff
@@ -30,18 +31,20 @@
 </script>
 
 <nav
```

Comment:
```suggestion

<svelte:window on:resize={() => dispatch("toggle", false)} />

<nav
```

you can add this line. Otherwise, when mobilenav is open and user makes the window, the mobile nav is still open

Reply:
>  when mobilenav is open and user makes the window

? it's on mobile the window doesn't resize? or I'm missing something?